### PR TITLE
Group actions and system message update

### PR DESCRIPTION
**Pull request by ibencetic-CS**
[Task #1529](https://app.productive.io/9193-clover-studio/tasks/6816815)
[Task #1526](https://app.productive.io/9193-clover-studio/tasks/6811339)

### CODE REVIEW

File: public/management/index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
         />
 
         <title>Management</title>
-      <script type="module" crossorigin src="/management/assets/index-b62a1345.js"></script>
+      <script type="module" crossorigin src="/management/assets/index-695a22c6.js"></script>
```
Comment (jklimpf) [inline public/management/index.html:29]: Why is this changed?

Comment (ibencetic-CS) [reply public/management/index.html:29]: This is the only tracked file from the `public/management` folder. I didn't change it manually. Maybe it changes when you rebuild?

**Review by jklimpf — APPROVED** [review]
